course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -433,6 +445,17 @@ def test_include_retired_programs(self):
             NestedProgramSerializer([retired_program], many=True, context=self.serializer_context).data
         )
 
+    def test_filter_course_runs_on_edx_org_short_name(self):
+        """
+        Verify courses runs filtering on edX organization.
```

Comment:
nit:
`Verify that course runs filtering on edX organization works as expected.`

Reply:
Done